pkg/manager-server/file_system_lustre.go

```diff
@@ -148,14 +148,16 @@ func (f *FileSystemLustre) Delete() error {
 	var err error
 	if f.Oem.BackFs == BackFsZfs {
 		zpool := f.zfsPoolName()
+
 		// Query the existence of the pool.
 		_, err = f.run(fmt.Sprintf("zpool list %s", zpool))
-		if err != nil {
-			return err
-		}
-		_, err = f.run(fmt.Sprintf("zpool destroy %s", zpool))
-		if err != nil {
-			return err
+
+		// If no error, delete. Or if error found and it's something other than 'no such pool' error, delete.
```

Comment:
Consider the positive form of this comment:
"If the pool still exists, attempt to delete it."